add-new-exercise.ps1

```diff
@@ -0,0 +1,51 @@
+param (
+    [Parameter(Mandatory=$true)][string]$Exercise,
+    [Parameter()][string[]]$Topics = $null,
+    [Parameter()][bool]$Core,
+    [Parameter()][int32]$Difficulty = 1,
+    [Parameter()][String]$UnlockedBy = $null
+ )
+
+class Exercise {
+    [String]$slug = ""
+    [guid]$uuid = [Guid]::Empty
+    [Boolean]$core = $false
+    [string]$unlocked_by = $null
```

Comment:
```suggestion
    [string]$unlockedBy = $null
```

Reply:
By using `unlocked_by` instead of `unlockedBy ` we are achieving the correct json output without adding any other manipulation. If you agree I suggest we keep the property with the underscore in this context

Reply:
Makes sense.